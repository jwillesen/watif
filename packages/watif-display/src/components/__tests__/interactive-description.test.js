import React from 'react'
import {render} from 'react-testing-library'
import InteractiveDescription from '../interactive-description'

it('renders properly', () => {
  const {getByText} = render(
    <InteractiveDescription
      emptyText="nothing here"
      watext={<text>description here</text>}
      verbs={[]}
      onItemClick={jest.fn()}
      onVerbClick={jest.fn()}
    />
  )
  getByText('description here')
})
