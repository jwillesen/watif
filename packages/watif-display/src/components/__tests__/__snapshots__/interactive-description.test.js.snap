// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<div
  className="root"
>
  <h2
    className="heading"
  />
  <div
    className="content"
  >
    <div>
      <Watext
        onItemClick={[MockFunction]}
        watext={
          <text>
            description here
          </text>
        }
      />
    </div>
    <div
      className="verbs"
    >
      <VerbBar
        onVerbClick={[MockFunction]}
        verbs={Array []}
      />
    </div>
  </div>
</div>
`;
